modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -1809,6 +1840,265 @@ paths:
+                  required: true
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+            tags: ["ObjectModel"]
+        put:
+            operationId: putObjectModelByExternalReferenceCode
+            parameters:
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+                    application/xml:
+                        schema:
+                            $ref: "#/components/schemas/ObjectModel"
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/ObjectModel"
+            tags: ["ObjectModel"]
+    "/object-models/by-external-reference-code/{externalReferenceCode}/object-definitions":
+        get:
+            operationId: getObjectModelByExternalReferenceCodeObjectDefinitionsPage
+            parameters:
+                - in: path
+                  name: externalReferenceCode
+                  required: true
+                  schema:
+                      type: string
+                - in: query
+                  name: page
+                  schema:
+                      type: integer
+                - in: query
+                  name: pageSize
+                  schema:
+                      type: integer
+                - in: query
+                  name: search
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/ObjectDefinition"
+                                type: array
+                        application/xml:
+                            schema:
+                                items:
+                                    $ref: "#/components/schemas/ObjectDefinition"
+                                type: array
+            tags: ["ObjectDefinition"]
+    "/object-models/{objectModelId}":
```

Comment:
For now, let's avoid exposing endpoints that uses Id as reference. Just the ERC are needed here.
Do you see any need to have those?

Reply:
We will await Marco's guidance.